tests/testthat/setup-workflow-ref.R

```diff
@@ -194,7 +194,16 @@ create_rlg_models <- function() {
   )
 }
 
-cleanup <- function() {
+clean_test_enviroment <- function(.f, env = parent.frame())
+{
+  cleanup(env)
+  .f()
+  withr::defer(cleanup(env), envir = env)
+}
+
+
+
+cleanup <- function(env = parent.frame()) {
```

Comment:
Now that you're passing in `env`, the `rm` calls downstream should pass that as `pos`.